modules/distribution-zones/src/main/java/org/apache/ignite/internal/distributionzones/DistributionZoneManager.java

```diff
@@ -798,11 +412,11 @@ private void createOrRestoreZoneState(DistributionZoneView zone, long revision)
 
             ZoneState prevZoneState = zonesState.putIfAbsent(zoneId, zoneState);
 
-            assert prevZoneState == null : "Zone's state was created twice [zoneId = " + zoneId + ']';
+            assert prevZoneState == null : "Zone's state was created twice: zoneId=" + zoneId;
```

Comment:
Probably I've missed that. We should not use square braces any longer? Same is about spaces before and after '='.

Reply:
returned it(